function.js

```diff
@@ -1,77 +1,43 @@
-}
-
-function myWork() {
-  document.getElementById("home").style.visibility = "hidden";
-  document.getElementById("opening").style.visibility = "hidden";
-  document.getElementById("skillset").style.visibility = "hidden";
-  document.getElementById("work").style.visibility = "visible";
-  document.getElementById("education").style.visibility = "hidden";
-  document.getElementById("interest").style.visibility = "hidden";
-  document.getElementById("reason").style.visibility = "hidden";
-  document.getElementById("misc").style.visibility = "hidden";
-}
-
-function myStudy() {
-  document.getElementById("home").style.visibility = "hidden";
-  document.getElementById("opening").style.visibility = "hidden";
-  document.getElementById("skillset").style.visibility = "hidden";
-  document.getElementById("work").style.visibility = "hidden";
-  document.getElementById("education").style.visibility = "visible";
-  document.getElementById("interest").style.visibility = "hidden";
-  document.getElementById("reason").style.visibility = "hidden";
-  document.getElementById("misc").style.visibility = "hidden";
-}
-
-function myHobby() {
-  document.getElementById("home").style.visibility = "hidden";
-  document.getElementById("opening").style.visibility = "hidden";
-  document.getElementById("skillset").style.visibility = "hidden";
-  document.getElementById("work").style.visibility = "hidden";
-  document.getElementById("education").style.visibility = "hidden";
-  document.getElementById("interest").style.visibility = "visible";
-  document.getElementById("reason").style.visibility = "hidden";
-  document.getElementById("misc").style.visibility = "hidden";
-}
-
-function myReason() {
-  document.getElementById("home").style.visibility = "hidden";
-  document.getElementById("opening").style.visibility = "hidden";
-  document.getElementById("skillset").style.visibility = "hidden";
-  document.getElementById("work").style.visibility = "hidden";
-  document.getElementById("education").style.visibility = "hidden";
-  document.getElementById("interest").style.visibility = "hidden";
-  document.getElementById("reason").style.visibility = "visible";
-  document.getElementById("misc").style.visibility = "hidden";
-}
-
-function myOthers() {
-  document.getElementById("home").style.visibility = "hidden";
-  document.getElementById("opening").style.visibility = "hidden";
-  document.getElementById("skillset").style.visibility = "hidden";
-  document.getElementById("work").style.visibility = "hidden";
-  document.getElementById("education").style.visibility = "hidden";
-  document.getElementById("interest").style.visibility = "hidden";
-  document.getElementById("reason").style.visibility = "hidden";
-  document.getElementById("misc").style.visibility = "visible";
-}
-
+function scrollingTop() {
+  const y = window.scrollY;
+  if(y > 0){
+    window.requestAnimationFrame(scrollingTop);
+    window.scrollTo(0, y - y / 30);
```

Comment:
What is `y - y / 30`? Avoid magic spells and maybe give the variable a name so that someone else, or yourself in the future can understand the intention at a glance.